### DISCUSSION TITLE
Example for hoomd.md.pair.Table with hoomd version 4.5.0

### DISCUSSION THREAD

**Discussion by Dipanjali-1993** [glotzerlab/hoomd-blue#1733] [Support] [answered]
I have found an example for the class hoomd.md.pair.table with the hoomd version 2.9.7. However, can you please provide an example for the same class  hoomd.md.pair.table with the latest hoomd version 4.5.0. I don't find any example in the documentation for that class. 

If I define U = 4 * epsilon * ( (sigma / r)**12 - (sigma / r)**6) and  F = 4 * epsilon / r * ( 12 * (sigma / r)**12 - 6 * (sigma / r)**6) with rmin=0.8, rmax=3.0, epsilon=1.5 and sigma=1.0.  Then how to use the class  hoomd.md.pair.table  for the function U and F with the latest hoomd version 4.5.0.

**Comment by joaander** [comment, marked as answer]
For a U with that functional form, I recommend that you use `hoomd.md.pair.LJ`.
